### DISCUSSION TITLE
Multi-account support

### DISCUSSION THREAD

**Discussion by hetchel** [sgkoishi/PackTracker#30] [Ideas]
I have more than one account.
I managed to get it working by closing and renaming the "Pack Tracker" data folder between several ones.
Could this be considered for a future feature (there really SEEMS very easy to implement ;)

**Comment by sgkoishi** [comment]
It came up with some other questions like:
* How to detect account switch? 
  * BattleTag?
    * BattleTag is not permanent, so different tags may point to the same player. 
      * Should a message box shows up when a BattleTag change is detected, asking if switching to a new profile is required?
    * Even worse, same BattleTag may point to multiple accounts in multiple servers I guess?
      * Someone register in America zone with Tag#1234 and they can enter EU/Asia region(server) keeping their tag but with no game progress 
* How to view data for different accounts?
  * Of course we can only show the data of current account, and ask the users to log in to the corresponding account to view statistics

**Reply by hetchel** [reply]
My take:
Battletag is ok. Battletag change is not so much frequent. If it's clearly explained that we don't do it or asked as you propose, it will be more than ok. If you create a different data folder for each account "Packtracker_Battletag", it would be easily fixed either automatically or manually.
For those who are using the same account in multiple servers the extension is already not working for them (but I'm all for trying to provide a solution for them).
You view the data of the last account or the current account, or only for the current account if the former is not easy. There is already data that change for logged/non-logged (number of non-tracked packs). For pack opening (the problematic step) you have to be logged, so....
